source/release-notes/2.2.0.rst

```diff
@@ -16,6 +16,10 @@ For install instructions, see :ref:`deploy-pmm`.
 Improvements and new features
 =============================
 
+* :pmmbug:`4907` and :pmmbug:`4767`: The user's AMI Instance ID is now used
+  to facilitate adding an AWS RDS MySQL or Aurora MySQL Instance for monitoring,
+  which provides both additional verification and automatic discovery of
+  instances.
```

Comment:
this   issue is only about  AMI  installation, not  Adding RDS  instances for monitoring